scalactic-test/src/test/scala/org/scalactic/DirectRequirementsSpec.scala

```diff
@@ -4794,7 +4794,7 @@ class DirectRequirementsSpec extends FunSpec with OptionValues {
 
     it("should throw NullArgumentException with correct message when one of passed parameters through object method call is null") {
       class AClass {
-        def returnNull: String = null
+        def returnNull: String | Null = null
       }
       val aClass = new AClass
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.